iconservice/icx/issue/issue_engine.py

```diff
@@ -41,17 +46,20 @@ def close(self):
             self._storage.close(context=None)
             self._storage = None
 
+        if self._issue_regulator:
+            self._issue_regulator.close()
+
     def _issue(self,
                context: 'IconScoreContext',
                to: 'Address',
                amount: int):
-        assert amount > 0
+        if amount == 0:
```

Comment:
remove if statement

Reply:
I got a mistake. have removed this statement